tests/async.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { StrictMode, Suspense, useEffect, useRef } from 'react'
+import React, { StrictMode, Suspense, useEffect, useRef } from 'react'
```

Comment:
It's our intention to prefer jsx-transform. If swc doesn't support it, can we somehow hack it without changing the code please?